### PR TITLE
Fix description logic 1

### PR DESCRIPTION
**Pull request by mattiasostmar**
It seems the check for OK_to_upload needs to be at the end.

### CODE REVIEW

**Comment by mattiasostmar** [comment]
Now the description field includes Händelse / var närvarande vid and includes Ort, foto even though the field Beskrivning is empty.

**Comment by mattiasostmar** [comment]
And heres the gist with the example image: https://gist.github.com/mattiasostmar/f9c9901817547ab1d6c8632e9fab6a73

File: 1. Mexiko Create filenames filename-mappings and infofiles.py [hunk 1]
```diff
@@ -300,13 +276,10 @@ def create_infofiles(row, filenames_file, not_ok_file):
     en_description = "{{en|Images from the 1932 Sigvald Linné archeological expedition at Teotihuacán, Mexico.}}\n"
     
     # Maintanance category and/or skip
```
Comment (lokal-profil) [inline 1. Mexiko Create filenames filename-mappings and infofiles.py]: this line should also be moved

**Review by lokal-profil — CHANGES_REQUESTED** [review]
I don't see why this should have changed the behaviour but if it works then great =)

One minor edit suggested inline.

Check that a few known non-OK_to_upload files are still not being outputted (just in case)

File: 1. Mexiko Create filenames filename-mappings and infofiles.py (hunk 1 above)
Comment (lokal-profil) [reply 1. Mexiko Create filenames filename-mappings and infofiles.py]: i.e. the comment

**Comment by mattiasostmar** [comment]
I checked 4 Not_OK_to_upload files and they are not being outputted.

**Comment by lokal-profil** [comment]
perfect =)

**Pull request merged**